Sprint-1/exercises/count.js

```diff
@@ -4,3 +4,5 @@ count = count + 1;
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
 // Describe what line 3 is doing, in particular focus on what = is doing
+
+// Line 3 is adding 1 to count. After this line, the value of count is 1.
```

Comment:
In Line3, 'count = count + 1' is assignment statement which increments the value of count by 1.
 '='  is the assignment operator which assigns the value of the right statement to the variable on the left side.
First, the operator evaluates the value of count+1 which is 1 and since we have the '=' symbol, it assigns this value to count which initially was 0.